### PR TITLE
Add set defaults tx info

### PR DESCRIPTION
**Pull request by hansl**
And new tests.

### CODE REVIEW

File: src/many/src/types/ledger.rs [hunk 1]
```diff
@@ -426,11 +431,76 @@ impl Transaction {
...
+macro_rules! encode_tx_info {
+    ( $( $sname: ident { $( $idx: literal => $name: ident : $type: ty, )* }, )* ) => {
+        impl Encode for TransactionInfo {
+            fn encode<W: encode::Write>(
+                &self,
+                e: &mut Encoder<W>,
+            ) -> Result<(), encode::Error<W::Error>> {
+                match self {
+                    $(  TransactionInfo :: $sname { $( $name, )* } => {
+                            e.map( 1u64 $(+ replace_expr!($idx 1u64))* )?
```
Comment (fmorency) [inline src/many/src/types/ledger.rs:457]: This doesn't seem to fit what's in the spec in PR https://github.com/many-protocol/specification/pull/35/files

`{ 0 => TransactionKind::foo, 1 => foo@param }`

Also, I think the naming should be revised; there's a `ledger-transaction-type-send` in the spec, but here it's a `TransactionKind`, `ledger-transaction-info` vs `account-multisig-transaction` which are both `TransactionInfo`, but not really.

**Review by fmorency — COMMENTED** [review]
Some comments.

File: src/many/src/server/module/_6_ledger_commands.rs [hunk 2]
```diff
@@ -7,5 +8,5 @@ pub use send::*;
 
 #[many_module(name = LedgerCommandsModule, id = 6, namespace = ledger, many_crate = crate)]
 pub trait LedgerCommandsModuleBackend: Send {
-    fn send(&mut self, sender: &Identity, args: SendArgs) -> Result<(), ManyError>;
+    fn send(&mut self, sender: &Identity, args: SendArgs) -> Result<EmptyReturn, ManyError>;
```
[the remark this answers is not in the document]
Comment (hansl) [reply src/many/src/server/module/_6_ledger_commands.rs:11]: Done. For the record, it's not a new type but a type alias. We should probably move to new types for those but it's a lot of work we don't need right now.

File: src/many/src/types/identity.rs [hunk 3]
```diff
@@ -134,20 +134,38 @@ impl Identity {
     }
 }
 
+impl Identity {
```
[the remark this answers is not in the document]
Comment (hansl) [reply src/many/src/types/identity.rs:137]: Sometimes I need a break, I don't know what I was thinking. I've fixed it with `#[cfg(any(test, feature = "raw"))]`.

File: src/many/src/types/ledger.rs (hunk 1 above)
Comment (hansl) [reply src/many/src/types/ledger.rs:495]: Let's discuss? I think you got the wrong thing or maybe outdated.

**Review by fmorency — APPROVED** [review]

**Pull request merged**